src/schema/schema_create.c

```diff
@@ -311,6 +311,12 @@ __create_file(WT_SESSION_IMPL *session, const char *uri, bool exclusive, const c
         WT_ERR(__wt_session_release_dhandle(session));
 
 err:
+    if (import) {
+        if (ret != 0)
+            WT_STAT_CONN_INCR(session, session_table_create_import_fail);
+        else
+            WT_STAT_CONN_INCR(session, session_table_create_import_success);
+    }
```

Comment:
I'm confused about what we want to count.
1. Number of imported files **success/fail**
2. Number of WT_SESSION::create("import") calls **success/fail**
3. Number of **successfully** imported files. And number of **fails** of WT_SESSION::create("import")
4. Number of **successfully** imported files. And number of WT_SESSION::create("import") calls **success/fail**

In my opinion, option 4 is the most informative as it keeps track of the number of import API calls and imported files. What do you think @sueloverso ?

Reply:
I think option 2 is the easiest to reason about and keeps the code the cleanest. It is a subset of the `WT_SESSION::create` statistics. The user has no concept of "number of files" and I am not entirely sure what that means. The `create` API takes the URI of the table. Whether that call results in some number of objects (tiered) or chunks (LSM) or whatever for the table is not relevant to the user I think.

Reply:
That being said, if option 2 is all we're counting then the above code should be all we need.